tutorials/nservicebus-101/lesson-3/tutorial.md

```diff
@@ -0,0 +1,203 @@
+
+In fact, you will probably get a giant wall of exception text, because the message is tried and retried, and then retried some more after successively longer delays, until finally failing for good some time later. We'll cover this behavior in more detail in [Lesson 5: Retrying errors](../lesson-5/).
+
+The important part is, this is a good thing! If we accidentally send a message to an endpoint we didn't intend, it won't just fail silently, lost forever to the nothingness of the great beyond.
+
+
+### Sending to another endpoint
+
+Now we need to fix up the ClientUI so that it is sending `PlaceOrder` to the Sales endpoint.
+
+ 1. In the **ClientUI** endpoint, modify the **Program.cs** file so that `endpointInstance.SendLocal(command)` is replaced by `endpointInstance.Send(command)`.
+ 1. In the `AsyncMain` method of the same file, change the call to use the MSMQ transport to access the routing configuration and specify the logical routing for `PlaceOrder` by changing your code as follows.
+
+snippet:AddingRouting
+
+This establishes that commands of type `PlaceOrder` should be sent to the **Sales** endpoint. While these two calls could be strung together in the same statement, there will commonly be multiple logical routes specified, so it's advantageous to assign the `routing` variable for future use.
+
+
+### Running the solution
+
+Now when we run the solution, we get two console windows, one for ClientUI and one for Sales. After moving them around so that we can see both, we can try to place an order by pressing `P` in the **ClientUI** window.
+
+INFO: You can also keep console windows from showing up in random screen locations each time by right-clicking the console window's title bar, and in the **Layout** tab, unchecking the **Let system position window** checkbox.
+
+In the **ClientUI** window, we see this output:
+
+```no-highlight
+INFO  ClientUI.Program Press 'P' to place an order, or 'Q' to quit.
+p
+INFO  ClientUI.Program Sending PlaceOrder command, OrderId = af0d1aa7-1611-4aa0-b83d-05e2d931d532
+INFO  ClientUI.Program Press 'P' to place an order, or 'Q' to quit.
+p
+INFO  ClientUI.Program Sending PlaceOrder command, OrderId = e19d6160-595a-4c30-98b5-ea07bc44a6f8
+INFO  ClientUI.Program Press 'P' to place an order, or 'Q' to quit.
+```
+
+Everything is the same, except the command is not processed here.
+
+In the **Sales** window, we see this:
+
+```no-highlight
+Press Enter to exit.
+INFO  Sales.PlaceOrderHandler Received PlaceOrder, OrderId = af0d1aa7-1611-4aa0-b83d-05e2d931d532
+INFO  Sales.PlaceOrderHandler Received PlaceOrder, OrderId = e19d6160-595a-4c30-98b5-ea07bc44a6f8
+```
+
+Let's take a second and reflect upon what we've achieved. We've managed to create two processes and achieve inter-process communication between them. Compared to the amount of effort required to set up and configure a WCF service, NServiceBus is able to communicate between processes with amazing ease!
```

Comment:
>  Compared to the amount of effort required to set up and configure a WCF service, NServiceBus is able to communicate between processes with amazing ease!

remove this. dropping in a comparison to WCF (and a cheap shot at that) adds no value